common/views/components/Modal/Modal.js

```diff
@@ -35,6 +35,7 @@ const Overlay = styled.div`
 `;
 
 const CloseButton = styled(Space).attrs({
+  'data-test-id': 'close-modal-button',
```

Comment:
Should this be a role? I don't think we should be using this for everything, just rather where we have no other way.

[Here's a good article on it](https://testing-library.com/docs/guide-which-query/)

Reply:
That makes a lot of sense. Thanks.